lib/modules/manager/leiningen/__fixtures__/parent.clj

```diff
@@ -0,0 +1,12 @@
+(defproject org.example/parent-project "1.0.0-SNAPSHOT"
```

Comment:
please inline, search `codeBlock` for samples

Reply:
https://github.com/renovatebot/renovate/pull/29552/commits/2b19582c91593616a46f881208f29bc38162570e